tests/components/eafm/conftest.py

```diff
@@ -0,0 +1,30 @@
+"""eafm fixtures."""
+# pylint: disable=redefined-outer-name
+
```

Comment:
We don't need this if we use the pytest fixture name parameter and name the fixtures to something different than the function name.
```suggestion
```